slack-message.h

```diff
@@ -6,7 +6,12 @@
 #include "slack-object.h"
 
 gchar *slack_html_to_message(SlackAccount *sa, const char *s, PurpleMessageFlags flags);
+void add_slack_attachments_to_buffer(GString *buffer, SlackAccount *sa, json_value *attachments, PurpleMessageFlags *flags);
```

Comment:
These should all be static, since they're not exported (and some have unqualified names).